### DISCUSSION TITLE
Hyprfocus + ultrawide monitors

### DISCUSSION THREAD

**Discussion by chriselrod** [hyprwm/Hyprland#5461] [Q&A]
I have an ultrawide monitor, where it is generally nice to have 6 or so equal width columns side by side.

The default dwindle layout is only good for up to 4 columns, or at least only does powers of 2.

For this I've started using the hy3 plugin.

However, hy3 doesn't appear to be compatible with hyprfocus, making it a little more difficult to figure out where the focus is when switching workspaces.
That is, when I've enabled both plugins, I know longer get hyprfocus's flash or shrink animations when changing focus, without any other indicators of a problem.

Are there alternative columnar layouts to hy3, that are compatible (I'm not looking for anything from hy3 other than the equal-width column splits)?

Or, what would it take to update/write a plugin for this?
I'm comfortable with C++, but not with Wayland or Hyprland's codebases/APIs.

Thought I'd ask here if there are existing solutions I'm not aware of before taking a deeper look.

**Comment by ozturkkl** [comment]
I also want to know about this! In the same boat. The master layout almost takes me there with nice gutter on the sides thing they agreed to the center position when only the master is up with no slaves. 

But I also got bit by column count not being enough. I was literally exploring hy3 today. 

@chriselrod I know you didn't get an answer to your question here but I'd really appreciate it you share with me what you ended up settling with since I'm in the same boat...

**Reply by chriselrod** [reply]
@ozturkkl I settled on niri and am very happy with it.
